scripts/docs/snippetadapter.js

```diff
@@ -549,6 +556,86 @@ function countUniqueSnippets( snippets ) {
 	return new Set( Array.from( snippets, snippet => snippet.snippetName ) ).size;
 }
 
+/**
+ * Returns a configuration for webpack that parses the `/docs/_snippets/assets.js` file.
+ * Thanks to that, we're able to load libraries from the `node_modules` directory in our snippets.
+ *
+ * @param {Object} config
+ * @param {Boolean} config.production Whether to build for production.
+ * @param {Object} config.snippetWebpackConfig The configuration returned by the `getWebpackConfig()` function.
+ * It is used to configure the output path for the asset file.
+ * @returns {Object}
+ */
+function getWebpackConfigForAssets( config ) {
+	return {
+		mode: config.production ? 'production' : 'development',
+
+		entry: {
+			assets: path.join( __dirname, '..', '..', 'docs', '_snippets', 'assets.js' )
+		},
+
+		output: {
+			filename: '[name].js',
+			path: config.snippetWebpackConfig.output.path
+		},
+
+		optimization: {
+			minimizer: [
+				new TerserPlugin( {
+					sourceMap: true,
+					terserOptions: {
+						output: {
+							// Preserve CKEditor 5 license comments.
+							comments: /^!/
+						}
+					},
+					extractComments: false
+				} )
+			]
+		},
+
+		plugins: [
+			new MiniCssExtractPlugin( { filename: '[name].css' } ),
+			new webpack.BannerPlugin( {
+				banner: bundler.getLicenseBanner(),
+				raw: true
+			} ),
+			new ProgressBarPlugin( {
+				format: 'Building assets for snippets: :percent (:msg)'
+			} )
+		],
+
+		// Configure the paths so building CKEditor 5 snippets work even if the script
+		// is triggered from a directory outside ckeditor5 (e.g. multi-project case).
+		resolve: {
+			modules: [
+				...getPackageDependenciesPaths(),
+				...getModuleResolvePaths()
+			]
+		},
+
+		resolveLoader: {
+			modules: getModuleResolvePaths()
+		},
+
+		module: {
+			rules: [
+				{
+					test: /\.svg$/,
+					use: [ 'raw-loader' ]
+				},
```

Comment:
Are we going to use this loader? It doesn't seem necessary so far (or am I missing something?).

Reply:
@pomek can you provide your expertise? :)

Reply:
ATM it loader can be deleted.

Reply:
Fixing it now, thanks @pomek!